app/views/layouts/application.html.erb

```diff
@@ -23,6 +23,16 @@
     <%= render 'header/header' %> 
     <section>
       <%= yield %>
+        <p class="navbar-text float-right">
```

Comment:
_note:  The PR is in the `devise` branch. However, the review and review comments are based on the `dev` branch, as requested by the student._

# Backend review

- [ ]  You installed Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ] Your app has no tests, kindly add some in order to ensure your app's integrity.

- [ ] Your backend project does not contain a readme file.  This is a hard requirement for all Microverse projects.  Therefore, kindly go [here](https://github.com/microverseinc/readme-template) and use this template.

Kindly remember that all sections are required.